src/pages/journey/[id].jsx

```diff
@@ -0,0 +1,87 @@
+import { Fragment, useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { Meta } from "../../layout/Meta.tsx";
+import AppPageTwoColumn from "../../layout/AppPageTwoColumn";
+import Loading from "../../components/Loading";
+import ErrorMessage from "../../components/ErrorMessage";
+import SplashHeader from "../../templates/LearningJourney/SplashHeader";
+import Details from "../../templates/LearningJourney/Details";
+import CurriculumSidebar from "../../templates/LearningJourney/CurriculumSidebar";
+import { withPageAuthRequired } from "@auth0/nextjs-auth0";
+import { GET_USERS, GET_LEARNING_JOURNEY } from "../../lib/graphql";
+import { useQuery } from "@apollo/client";
+import ContentView from "../../templates/LearningJourney/ContentView";
+
+const LearningLandingPage = () => {
+  const router = useRouter();
+  const { id, bit } = router.query;
+  const [currentBitId, setCurrentBitId] = useState(null);
+  const [started, setStarted] = useState(false);
+  const [userLearningJourneyData, setUserLearningJourneyData] = useState(null);
+  const { loading, error, data } = useQuery(GET_USERS);
+  const { data: learningJourneyDataArray } = useQuery(GET_LEARNING_JOURNEY, {
+    variables: { learningJourneyId: id },
+  });
+  const user = data?.users[0];
+  const learningJourneyData = learningJourneyDataArray?.learningJourney[0];
+
+  useEffect(() => {
+    if (bit) {
+      setCurrentBitId(bit);
+      setStarted(true);
+    }
+  }, [bit]);
+
+  const handleStart = () => {
+    setStarted(true);
+  };
+
+  return (
+    <div className="h-full">
+      {!learningJourneyData ? (
+        <Loading />
+      ) : (
+        <AppPageTwoColumn
+          meta={
+            <Meta
+              title={learningJourneyData.title}
+              description={learningJourneyData.description}
+            />
+          }
+          leftColumn={
+            loading ? (
+              <Loading />
+            ) : !started ? (
```

Comment:
Should not we have to check in the learning journey if it's started or not, instead of clicking the start this journey every time we load the page?

Reply:
I think the splash page is going to act as a way to share an overview of the learning journey, including new learning reflections and updates from the guide.

We can monitor and get user feedback. If the splash page is not helpful then yes it would make more sense to go to the learning journey directly if previously started.